index.html

```diff
@@ -751,8 +783,8 @@ <h3 id="configuration">Configuration</h3>
       </tr>
       <tr id="input-value">
         <td><b>inputValue</b></td>
-        <td><i>''</i></td>
-        <td>Input field initial value.</td>
+        <td><i>''<br>Promise</i></td>
```

Comment:
This is the default value, it's still the empty string, no need in mentioning Promise here.